httpx/_models.py

```diff
@@ -1577,7 +1577,7 @@ def iter_bytes(self, chunk_size: int = None) -> typing.Iterator[bytes]:
             decoder = self._get_content_decoder()
             chunker = ByteChunker(chunk_size=chunk_size)
```

Comment:
Passing `chunk_size` through to `iter_raw()` *isn't* the right behaviour, no.
We're already handling the chunk_size correctly here.